apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -322,7 +366,7 @@ namespace MWPhysics
                 velocity = (osg::Quat(refpos.rot[2], osg::Vec3f(0, 0, -1))) * movement;
 
                 if ((velocity.z() > 0.f && physicActor->getOnGround() && !physicActor->getOnSlope())
-                 || (velocity.z() > 0.f && velocity.z() + inertia.z() <= -velocity.z() && physicActor->getOnSlope()))
+                 || (velocity.z() > 0.f && velocity.z() + inertia.z() <= -velocity.z() && physicActor->getOnSlope())) // note: this should be less tolerant now that it's harder to get stuck in acute seams
```

Comment:
I'm concerned about this hack of mine because false-positive situations it (being a temporary solution) can cause are not good, and if anything, its usage should be minimized.

Reply:
It's still important since it's still pretty easy to get stuck with capsules, but it should probably be replaced with something that's harder to trigger. Maybe based on lack of travel rather than velocity. Might be a good target for a followup PR.

Reply:
Well, it's not critical I guess.